services/pane.test.js

```diff
@@ -156,54 +157,108 @@ describe(dirname, function () {
       });
     });
 
-    describe('openValidationErrors', function () {
-      var fn = lib[this.title],
-        sandbox;
+    describe('openNewPage', function () {
+      var mock = {
+          locals: {edit: true}
+        },
+        noFilter = function () {},
+        createPage, el, env, getState, pagePane, result, sandbox, stub, template;
 
       before(function () {
-        var template = document.createElement('template'),
-          headEl = document.createElement('div'),
-          paneInnerEl = document.createElement('div'),
-          toolbar = document.createElement('div');
-
-        // add template
-        template.classList.add('kiln-pane-template');
-        headEl.classList.add('pane-header');
-        paneInnerEl.classList.add('pane-inner');
-        template.content.appendChild(headEl);
-        template.content.appendChild(paneInnerEl);
-        document.body.appendChild(template);
-
-        // add toolbar
-        toolbar.classList.add('kiln-toolbar');
-        document.body.appendChild(toolbar);
+        env = new nunjucks.Environment();
+        // satisfy request for nunjucks filter in template
+        env.addFilter('includeFile', noFilter); // TODO – include nunjucks filters
+        template = env.getPreprocessedTemplate('template.nunjucks');
       });
 
       beforeEach(function () {
+        result = template.render(mock);
         sandbox = sinon.sandbox.create();
         sandbox.stub(ds);
-        sandbox.stub(state, 'get');
+        getState = sandbox.stub(state, 'get');
```

Comment:
now that `state.get` was removed from the code, I think it can be removed from the related tests as well.